python/tvm/relax/frontend/onnx/onnx_frontend.py

```diff
@@ -637,6 +637,11 @@ def _impl_v14(cls, bb, inputs, attr, params):
         x = inputs[0]
         k = inputs[1] if len(inputs) > 1 else 0
 
+        if isinstance(k, relax.Var) and k.name_hint in params:
+            k = get_constant(k, params)
+        else:
+            k = 0
```

Comment:
Why do we need to set `k` to 0? It should already be done in line 638

Reply:
Also, what will happen if `k` is a constant instead of an input? Please also add a test case for it.

Reply:
> Why do we need to set `k` to 0? It should already be done in line 638

In line 638 'k' is set to 0 if its value hasn't been passed. In line 643 we set it to 0 if 'k' is passed but it's value (constant) cannot be obtained.

> Also, what will happen if `k` is a constant instead of an input? Please also add a test case for it.

Thank you for the feedback! I will add a test case.